[2025-05-24T22:32:29.020+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-24T22:32:29.037+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.extract_api_data manual__2025-05-24T21:17:11.637733+00:00 [queued]>
[2025-05-24T22:32:29.041+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.extract_api_data manual__2025-05-24T21:17:11.637733+00:00 [queued]>
[2025-05-24T22:32:29.042+0000] {taskinstance.py:2303} INFO - Starting attempt 8 of 13
[2025-05-24T22:32:29.049+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): extract_api_data> on 2025-05-24 21:17:11.637733+00:00
[2025-05-24T22:32:29.057+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1844) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-24T22:32:29.058+0000] {standard_task_runner.py:63} INFO - Started process 1848 to run task
[2025-05-24T22:32:29.058+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'supplier_data_etl_pipeline', 'extract_api_data', 'manual__2025-05-24T21:17:11.637733+00:00', '--job-id', '151', '--raw', '--subdir', 'DAGS_FOLDER/supplier_dag.py', '--cfg-path', '/tmp/tmp7eisnnei']
[2025-05-24T22:32:29.060+0000] {standard_task_runner.py:91} INFO - Job 151: Subtask extract_api_data
[2025-05-24T22:32:29.092+0000] {task_command.py:426} INFO - Running <TaskInstance: supplier_data_etl_pipeline.extract_api_data manual__2025-05-24T21:17:11.637733+00:00 [running]> on host a7d211f7b093
[2025-05-24T22:32:29.153+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='supplier_data_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract_api_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-24T21:17:11.637733+00:00' AIRFLOW_CTX_TRY_NUMBER='8' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-24T21:17:11.637733+00:00'
[2025-05-24T22:32:29.154+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-24T22:32:29.157+0000] {etl.py:66} INFO - api data read
[2025-05-24T22:32:29.166+0000] {etl.py:68} INFO - api json data normalized
[2025-05-24T22:32:29.168+0000] {etl.py:70} INFO - api data source pickled
[2025-05-24T22:32:29.169+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-24T22:32:29.169+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-24T22:32:29.176+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=supplier_data_etl_pipeline, task_id=extract_api_data, execution_date=20250524T211711, start_date=20250524T223229, end_date=20250524T223229
[2025-05-24T22:32:29.199+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-24T22:32:29.212+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-24T22:32:29.213+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
